### PR TITLE
Topic slide

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

File: client/src/app/site/agenda/models/view-item.ts [hunk 1]
```diff
@@ -137,4 +138,17 @@ export class ViewItem extends BaseViewModel {
         }
         return false;
     }
+
+    public getSlide(): ProjectorElementBuildDeskriptor {
```
Comment (FinnStutzenstein) [inline client/src/app/site/agenda/models/view-item.ts]: If you want to project the content object, maybe a simple `return this.contentObject.getSlide()` is enough. Two things: Make shure, that contentObject is not null and that `isProjectable(this.contentObject)` is true.

Comment (FinnStutzenstein) [reply client/src/app/site/agenda/models/view-item.ts]: Whats about the case, if one of the assertions above fails? If you return null, make sure to catch this in the projector button, etc.

File: client/src/app/site/agenda/models/view-item.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { BaseViewModel } from '../../base/base-view-model';
 import { Item } from 'app/shared/models/agenda/item';
 import { Speaker } from 'app/shared/models/agenda/speaker';
 import { BaseAgendaViewModel, isAgendaBaseModel } from 'app/site/base/base-agenda-view-model';
+import { ProjectorElementBuildDeskriptor } from '../../base/projectable';
 
 export class ViewItem extends BaseViewModel {
```
Comment (FinnStutzenstein) [inline client/src/app/site/agenda/models/view-item.ts]: `extends baseProjectableViewModel`.

**Comment by emanuelschuetze** [comment]
@FinnStutzenstein All done. Please review again.

**Pull request merged**